flaskr/algorithm.py

```diff
@@ -52,7 +52,7 @@ def generate_profile(portfolio, finnhub_client):
         
         E = 19.23*(MB - 1.7364) + 0.0006*(Size + 4124.38)
         I = 18.518*(MB - 1.80656) + 0.0005*(12679.64 - Size)
-        S = 6.757*(2.37625 - MB) + 0.0295*(Size - 4895.96)
+        S = 13.514*(2.37625 - MB)
```

Comment:
just to clarify, this change was made bc Sentimentality is not associated with MB, right?

Reply:
The calculation gave equal weightage to Market-to-Book Ratio and Market Size, but this was producing skewed results due to issues with Market Size data. I figured MB is more consistent, so I gave MB full weightage

Reply:
also, can I ask why we need a cap for the Size at 14e3? (in line 50). thank you ^_^

Reply:
> 
> 
> also, can I ask why we need a cap for the Size at 14e3? (in line 50). thank you ^_^

Just for our convenience. Imagine a company like Amazon with huge market cap that would heavily skew the results. From my (unscientific) math, 14e3 seemed to be the limit. I'm open to changing this.

Reply:
> > also, can I ask why we need a cap for the Size at 14e3? (in line 50). thank you ^_^
> 
> Just for our convenience. Imagine a company like Amazon with huge market cap that would heavily skew the results. From my (unscientific) math, 14e3 seemed to be the limit. I'm open to changing this.

okay got it, I think we can just go with 14e3 for now then. thanks for explaining 👍